.buildkite/pipeline.build.yml

```diff
@@ -5,8 +5,8 @@
     - ./java/test.sh
 
 # the bulk of serve tests are now run on civ2 infra, and defined in
-# pipeline.build_serve.yml
-- label: ":serverless: Serve HA Tests"
+# serve.rayci.yaml
+- label: "serve civ1 tests"
```

Comment:
why was it `serverless`? that is kind of different, right?

Reply:
serverless is just the emoji; i guess they pick it because of the word 'serve'